### PR TITLE
Ext/hebrish

### PR DESCRIPTION
**Pull request by hushhash**
## Description

Fixed Hebrish to code review:
- Switched to no-view
- Added an extension icon
- Added try/catch where needed
- Cleaned up logs and prints

## Checklist

- [ ] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [ ] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [ ] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [ ] I checked that files in the `assets` folder are used by the extension itself
- [ ] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/hebrish/package.json [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  "icon": "hebrish-icon.png",
+  "author": "alon_shacham",
+  "categories": [
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "to-hebrew",
+      "title": "To Hebrew",
```
Comment (pernielsentikaer) [inline extensions/hebrish/package.json:15]:
```suggestion
      "title": "English to Hebrew",
```

File: extensions/hebrish/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+      "description": "Change Hebrew accidently written on an English keyboard to Hebrew",
+      "mode": "no-view"
+    },
+    {
+      "name": "to-english",
+      "title": "To English",
```
Comment (pernielsentikaer) [inline extensions/hebrish/package.json:21]:
```suggestion
      "title": "Hebrew to English",
```

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you:

- What about capital letters? It seems like when you convert to hebrew and back again it all small letters?

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

**Comment by hushhash** [comment]
Thank you for this!

Hebrew doesn't have capital letters.
The purpose of this extension is for when you try and write in English and
you accidentally typed it on a Hebrew keyboard or vice versa.
The better use case would be to write something in Hebrew, switch to
English using the function and then switch it back using the other function.
When you try and write capitalized English letters on a Hebrew keyboard, no
letters are actually being typed. Therefore, this is a limitation. It does
not however make the extension less useful for its intended purpose.
Makes sense?

All the best and thank you for this awesome platform.

--------------------
Alon Shacham

**Comment by pernielsentikaer** [comment]
Can we find another name than `To Hebrew` - Convert to Hebrew?

**Comment by hushhash** [comment]
Of course!
Do you want me to change them to "Convert to Hebrew" and "Convert to
English" and update the PR?
--------------------
Alon Shacham

**Comment by hushhash** [comment]
done
--------------------
Alon Shacham

**Comment by pernielsentikaer** [comment]
Hi @hushhash 👋

I'm really sorry that I didn't find the extension from the Store before reviewing this. Are you okay with using that, or enhancing it if something is missing?

**Pull request closed without merging**